.buildkite/pipeline.gpu_large.yml

```diff
@@ -57,5 +57,8 @@
     - cleanup() { if [ "${BUILDKITE_PULL_REQUEST}" = "false" ]; then ./ci/build/upload_build_info.sh; fi }; trap cleanup EXIT
     - DOC_TESTING=1 TRAIN_TESTING=1 TUNE_TESTING=1 ./ci/env/install-dependencies.sh
     - pip install -Ur ./python/requirements_ml_docker.txt
+    # (artur) Install torch version that also fits huggingface's usage so that the transformer GPU test passes
```

Comment:
huggingface themselves use 11.1 in their tests afaics. CI fails with a CUDA OOM.

Reply:
@krfricke